### ISSUE TITLE
Question about the support of other tasks

### ISSUE THREAD

**Issue by hsiaoyi0504** [MIC-DKFZ/nnUNet#436]
Hi there, thanks for providing such a great project.
I am curious about if it is able to handle other tasks or if there is a plan for supporting other tasks of the project.
In my case, we have pairs of 3D images and would like to predict one from another one.
It's somewhat similar to the segmentation task and the difference lies in the labels.
In the segmentation task, the labels are 1 and 0 for each voxel.
In my case, the labels are voxel values.

**Comment by FabianIsensee** [comment]
Hi,
unfortunately this is a very different task and cannot be covered by nnU-Net. You can of course use nnU-Net as a framework to implement it, but this may be difficult because nnU-Net revolves around segmentation masks being the target.
Best,
Fabian

**Comment by hsiaoyi0504** [comment]
Thanks. That pieces of information is still pretty useful.

**Comment by hsiaoyi0504** [comment]
@FabianIsensee Thanks for replying to me earlier. Would you mind directing me to which parts of the library are those for revolves around segmentation masks being the target? Thank you.

**Comment by FabianIsensee** [comment]
Hi,
essentially in every step there is something that revolves around the segmentation masks. Changing the task to be something else should be doable but you need to change a lot of small things in many different locations. Also you will need to retune some hyperparameters because you need a different loss function.
Best,
Fabian
